docs/modules/ROOT/pages/presets.adoc

```diff
@@ -44,3 +44,38 @@ NOTE: Class hashes were computed using {class-hash-cairo-version}.
 |===
 
 TIP: {starkli} class-hash command can be used to compute the class hash from a Sierra artifact.
+
+== Presets Usage
+
+:setup_project: xref:index.adoc#set_up_your_project[setting up a project]
+:install_lib: xref:index.adoc#install the_library[installing the Contracts for Cairo library]
+:presets_dir: link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.10.0/src/presets[presets directory]
+
+These preset contracts are ready-to-deploy which means they should already be declared on the Sepolia network.
```

Comment:
this is something we have failed to guarantee, we should probably add it to our development/release process. It's partially covered by our the PR process (also not fully applied) requiring that new features are tested onchain, but that's not necessarily true for the final versions that get released (just like a classhash might end up being different once merged to main)

Reply:
> this is something we have failed to guarantee, we should probably add it to our development/release process

Agreed. I added the step in our `Release process`

Reply:
In the future, I think it'd be nice to write a script for declaring the presets to make this even easier. Maybe we can leverage scarb's [scripts](https://docs.swmansion.com/scarb/docs/reference/scripts.html) for this e.g. `scarb run declare-presets`

Reply:
the hard part is to automate the payments, but yes